teams/teams-ps/teams/Set-TeamsEnvironmentConfig.md

```diff
@@ -0,0 +1,124 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Connect.dll-Help.xml
+Module Name: MicrosoftTeams
+online version:
+schema: 2.0.0
+author: VikneshMSFT
+ms.author: vimohan
+ms.reviewer: pbafna
+manager: vinelap
+---
+
+# Set-TeamsEnvironmentConfig
+
+## SYNOPSIS
+
+Sets environment specific configurations on local machine and will be used to connect to the right environment when running connect-microsoftteams
+
+## SYNTAX
+
+```
+Set-TeamsEnvironmentConfig [-EndpointUris <Hashtable>] [-TeamsEnvironmentName <String>] [-WhatIf] [-Confirm]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+This cmdlet sets environment specific configurations like endpoint URIs(AAD\Graph etc.) and Teams environment (GCCH\DOD etc) in local machine.  
+When running connect-microsoftteams, environment specific information set in this cmdlet will be considered unless overridden by parameters of connect-microsoftteams.
+Parameters passed to connect-microsoftteams will take precedence over the information set by this cmdlet. 
+This cmdlet is not applicable for public and Azure government cloud and was introduced specifically to support other special clouds of Microsoft.  
```

Comment:
the clouds are referred to in a particular way in the documentation, would suggest reframing for e.g.  Commercial, GCC High, GCC, DoD, etc. 

also nitpick - Connect-MicrosoftTeams

Reply:
What are those special cloud that this cmdlet should be run?

Reply:
These cmdlets are specific for GallatinChina and airgap clouds at this point. Can be extended for other new clouds at a later point.